qiskit/circuit/bit.py

```diff
@@ -54,14 +54,23 @@ def __init__(self, register=None, index=None):
 
     @property
     def register(self):
-        """Get bit's register."""
+        """Get the register of an old-style bit.
+
+        .. deprecated:: Qiskit Terra 0.17
```

Comment:
The directive `.. deprecated::` takes a single word as a parameter, typically the version number.  This is the current render otherwise:
<img width="806" alt="Screenshot 2023-03-02 at 15 32 09" src="https://user-images.githubusercontent.com/766693/222457761-f0b5feca-66f7-4a30-aa6c-6baba95fc53d.png">

```suggestion
        .. deprecated:: 0.17
```

Reply:
oh huh, thanks - I did not know that was how it worked.